### ISSUE TITLE
Collections, can it be sorted?

### ISSUE THREAD

**Issue by rekcodocker** [darktable-org/darktable#13728]
**Is your feature request related to a problem? Please describe.**

I keep the Collections on film roll, works fine. 
But the order seems random.  I don't understand the order.  
It would be nice if the list was sorted alphabetically on name, so I can quickly find something.
I always have to go through the whole list because the item I search can be anywhere.

Additional info:
I think I read there was some logic behind it, like the film-rolls are sorted file-path. 
However, although this might _explain_ why it is the way it is, it does not help me to _find_ something.

Preferred solution:
To have this list in alphabetical order.  Maybe not perfect but much preferable.

![image](https://user-images.githubusercontent.com/16322196/221174187-5813bcc5-28e3-4e8b-b18b-ef1896c14535.png)

**Comment by Nilvus** [comment]
This had already been discussed some time ago and change was a no. There's a logic behind that: filmroll order on your system. It's related to path and how path is done and import had been made. Filmroll can be ordered actually by number or folder. See hamburger menu, then preferences, of that module.

Then, to find something, you can just use field entry on top of the module and replace any string by string you want. Starting for example with "Wed" will just show you all your filmroll starting by Wed. You can also add joker like % (see tooltip when moving cursor above field entry). And remember that all this is explained on manual!

darktable already have quite many settings, probably too much. Adding too much settings just override it.

**Comment by Nilvus** [comment]
See: https://github.com/darktable-org/darktable/issues/6668

**Comment by rekcodocker** [comment]
Yes, I found the 'order by ID'. And order by 'Path'. To be honest I don't even know what an 'ID' is, to me it's an internal database thing which I don't want to know about.  I look at the names.

Neither helps me to find a film roll at a glance - I have to scan through the whole list.
And to be honest, a 'filter' function means that the list does not provide access at a glance.  

Perhaps I am using it wrong. 
For _me_ the purpose of the film roll list is to quickly find a film roll.  Since currently my item can be anywhere in the list, it does not provide that function.
But perhaps it's not supposed to do that.  Surprising but possible. 
Perhaps I am using it wrong and it's designed for something else.

**Comment by ptilopteri** [comment]
you can reduce the filmroll list using wildcards, ie:
  %SportCube%

your example is not English and I am lacking education.
